src/components/Header/Header.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react"
```

Comment:
The Header still needs more work. 

1. Spacing between nav items is not accurate
2. On mobile, the sidebar is covering up some of the page content.
3. The sidebar appears and disappears for a fraction of a second when you decrease the screen size.

I'm going to install a library to help you with building the navbar

Reply:
1. The spacing between both Cart, and Contact are accurate, but the input and the sign in buttons are different.